app/ts/McdaApp/Workspace/CurrentTab/Subproblem/ScaleRanges/ScaleRanges.tsx

```diff
@@ -21,11 +22,16 @@ export default function ScaleRanges() {
   return (
     <LoadingSpinner showSpinnerCondition={!filteredWorkspace}>
       <Grid container item xs={12}>
-        <Grid item xs={9} id="effects-table-header">
+        <Grid id="effects-table-header" item xs={9}>
           <Typography variant={'h5'}>
             <InlineHelp helpId="scale-ranges">Scale ranges</InlineHelp>
           </Typography>
         </Grid>
+        <Grid container item xs={3} justify="flex-end">
+          <ShowIf condition={!Boolean(warnings.length)}>
```

Comment:
this doesn't read as less complicated than `warnings.length === 0` to me with the extra negation

Reply:
Well.... I agree!